### ISSUE TITLE
Associations Doc Missing belongsToMany through details

### ISSUE THREAD

**Issue by VinGarcia** [cakephp/docs#5703]
**Issue Description**

The [Associations page][page] explains very well how to use most associations type but when explaining the ["belongsToMany" through option][option] it is not clear; It mentions we will need it when using pivot tables that have extra information (that's what I have in my code) but does not explain how to access this information.

Giving a more precise example supose I am accessing a belongsToMany relationship like this:

```php
// Slightly modified example by me:
$query = $products->find('all')->contain(['Stores']);
foreach ($query as $products) {
    echo $article->stores[0]->name;
}
```

How would I do to get an attribute of the pivot table called "selling_price"? This is not explained.

[page]: https://book.cakephp.org/3.0/en/orm/associations.html
[option]: https://book.cakephp.org/3.0/en/orm/associations.html#using-the-through-option

---

Since I am new to CakePHP I am not suitable to fix this issue by myself. Any help would be appreciated.

**Comment by VinGarcia** [comment]
I continued my search and I found [a topic][save-data] in the Saving Data doc page that seems to have an example close to what I needed:

[save-data]: https://book.cakephp.org/3.0/en/orm/saving-data.html#saving-additional-data-to-the-join-table

```php
$student->courses[0]->_joinData->grade = 80.12;
$student->courses[0]->_joinData->days_attended = 30;
$studentsTable->save($student);
```

Thus reading the property `$student->courses[0]->_joinData->grade` would be the correct way to access that property right?

In any case, maybe an example like this should be added on the belongsToMany through explanation, or at least a reference to other parts of the documentation that explains it.

Thanks again for any help.
